submissions/adeyemiJuwon/introductionToHtml/index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>marthas living brand</title>
+</head>
+<body>
+<header>
+    <div>
+        <input type="search" name="q" value="" aria-label="Search" placeholder="Search">
+            <ul>
+                <li>
+                    <a href="#">Log in</a>
+                </li>
+                <li>
+                    <a href="#">Create account</a>
+                </li>
+                <li>
+                    <a href="#">cart</a>
+                </li>
+            </ul>
+            
+    </div>
+    <div>
+        <a href="#">
+            <img src="img/mlogo.png" alt="">
+        </a>
+    </div>
+    <hr>
+    <nav>
+        <ul>
+            <li><a href="#">Best Sellers</a></li>
+            <li><a href="#">Baby Care &amp; Wear</a></li>
+            <li><a href="#">Women collection</a></li>
+            <li><a href="#">Electronic Accessiories</a></li>
+            <li><a href="#"> Lights &amp; Lighting</a></li>
+            <li><a href="#">Smart Home &amp; Garden</a></li>
+
+        </ul>
+    </nav>
+</header>
+    <hr>
+<section>
+    <div>
+        <h1>LAPTOP ACCESSORIES</h1>
+        <a href="">
+            <img src="img/laptop.jpg" alt="">
+        </a>
+        <p>Xiaomi Mi Notebook Pro 2019 15.6 inch Windows 10 Chinese Language Home Edition Intel Core i5 - 8250U Quad Core 1.6GHz CPU 8GB RAM 256GB SSD 1.0MP Front Camera Fingerprint Sensor</p>
+        <p> Regular price£1,485.83 Sale price£891.50 </p>    
+
+        <h4>Similar Product</h4>
+        <ol>
+            <li> <a href="#"> 11 Inch Laptop Bag Tablet Zipper Pouch Sleeve for MacBook Air</a></li>
+            <li> <a href="#">SSIMO#O 2 in 1 Business Style Laptop Bag Tablet Zipper Pouch Sleeve for MacBook 13 / 14 inch</a></li>
+            <li> <a href="#">SSIMOO 2 in 1 Business Style Laptop Bag Tablet Zipper Pouch Sleeve for MacBook 15 inch</a></li>
+            <li> <a href="#">SSIMOO S818 2 in 1 Dot Pattern Laptop Bag Tablet Zipper Pouch Sleeve for MacBook 14 / 15 inch</a></li>
+        </ol>
+    </div>
+    <div>
+        <h1>SMART HOME ACCESSORIES</h1>
+        <a href="">
+            <img src="img/light.jpg" alt="">
```

Comment:
Well done.
You are meant to use 2 product images . Check the criteria here 
https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/issues/7